Update installing-openvino-windows-header.md (#11221)
openvinotoolkit/openvino: docs/install_guides/installing-openvino-windows-header.md

@endsphinxdirective

> If you want to install Intel® Distribution of OpenVINO™ toolkit on Windows, you may use the following options to accomplish this: 

Reviewer: ```suggestion
If you want to install Intel® Distribution of OpenVINO™ toolkit on Windows, you have the following options: 
```

Author: Straight to the point :) Awesome.
